### PR TITLE
appending localization

### PR DESCRIPTION
**Pull request by JustinB1eber**
### What's done
cn local

### CODE REVIEW

File: save-frontend/src/main/resources/locales/cn/vulnerability.json [hunk 1]
```diff
@@ -14,10 +14,14 @@
...
+  "CWE IDs": "CWE编号",
   "Tags": "标签",
-  "References": "相关链接",
-  "COSV Submitter": "作者",
+  "Related Vulnerabilities": "Related Vulnerabilities",
+  "Confirm type": "确认类型",
+  "Related link": "相关链接",
+  "COSV Submitter": "提交者",
   "Organization": "组织",
   "Credits": "合作者",
```
Comment (DrAlexD) [inline save-frontend/src/main/resources/locales/cn/vulnerability.json:26]: Change "Credits" to "Collaborators"

File: save-frontend/src/main/resources/locales/cn/vulnerability.json [hunk 2]
```diff
@@ -14,10 +14,14 @@
...
   "Edit": "编辑",
   "Last update time": "上次更新时间",
-  "Details": "描述",
+  "Description": "描述",
+  "CWE Names": "CWE名称",
+  "CWE IDs": "CWE编号",
   "Tags": "标签",
-  "References": "相关链接",
-  "COSV Submitter": "作者",
+  "Related Vulnerabilities": "Related Vulnerabilities",
```
[the remark this answers is not in the document]
Comment (JustinB1eber) [reply save-frontend/src/main/resources/locales/cn/vulnerability.json:21]: yes, sure

File: save-frontend/src/main/resources/locales/cn/vulnerability.json (hunk 1 above)
Comment (JustinB1eber) [reply save-frontend/src/main/resources/locales/cn/vulnerability.json:26]: ok

**Comment by DrAlexD** [comment]
Please make localization for pages: Profile, Settings, Organizations and others.

**Comment by akuleshov7** [comment]
{
    "Add bio and info:": "添加介绍信息",
    "Upload or select avatar:": "上传或选择头像",
    "Save changes": "保存修改",
    "400 characters": "限400个字符",
    "Upload avatar": "上传头像",
    "profile": "简介",
    "Basic Settings": "基本设置",
    "Profile settings": "个人设置",
    "Login and email": "登录和邮箱",
    "Organizations": "组织",
    "Security Settings": "安全设置",
    "Personal access tokens": "个人访问token",
    "OAuth accounts": "认证账户",
    "Personal Statistics": "个人数据统计",
    "Delete Profile": "删除用户页"
}

**Review by akuleshov7 — APPROVED** [review]

File: save-frontend/src/main/resources/locales/cn/thanks-for-registration.json [hunk 3]
```diff
@@ -1,4 +1,6 @@
 {
   "Thank you for registration": "感谢您的注册",
+  "You need to wait until you will be reviewed and approved by admins": "很快我们的管理员会通过您的注册请求",
   "Your request is pending review": "当前此平台尚在建设中，我们的管理团队会及时查看您的需求和建议，并且尽快给予反馈。无需犹豫，欢迎随时联系我们:"
+
```
[the remark this answers is not in the document]
Comment (akuleshov7) [reply save-frontend/src/main/resources/locales/cn/thanks-for-registration.json]: fixed

**Pull request merged**